gif/README.md

```diff
@@ -1,5 +1,6 @@
 This folder contains all the metadata needed to produce the `.gif` files used
 in the root README.
 
-The program used to process the metadata is
-https://github.com/faressoft/terminalizer.
+The program uses asciinema to generate .cast files and renders them with
```

Comment:
```suggestion 
The program uses `asciinema` to generate `.cast` files and renders them with

Reply:
You mentioned `The program uses` but there's no program in the PR. Are you using some custom program that uses `asciinema`? if yes then mention the program also. If you are using `asciinema` to generate `.cast` files then mention it clearly.